### PR TITLE
Coreutils: added true, false and yes

### PR DESCRIPTION
**Pull request by Keyboard-Slayer**
It's possible that the build fail, if so we'll need to modify `applications/coreutils/.build.mk`

Those additions are according to the issue #162

### CODE REVIEW

File: applications/coreutils/true.cpp [hunk 1]
```diff
@@ -0,0 +1,9 @@
+#include <libsystem/Macros.h>
+
+int main(int argc, char **argv)
+{
+    __unused(argc);
+    __unused(argv);
+
+    return 1;
```
Comment (sleepy-monax) [inline applications/coreutils/true.cpp:8]: This should be return 0

**Review by sleepy-monax — CHANGES_REQUESTED** [review]

File: applications/coreutils/true.cpp (hunk 1 above)
Comment (Keyboard-Slayer) [reply applications/coreutils/true.cpp:8]: God damn it you're totally true

Comment (Keyboard-Slayer) [reply applications/coreutils/true.cpp]: @sleepy-monax It's fixed

File: applications/coreutils/yes.cpp [hunk 2]
```diff
@@ -0,0 +1,40 @@
+#include <libsystem/io/Stream.h>
+#include <libutils/StringBuilder.h>
+
+String concat(int argc, char **argv)
+{
+    int i;
+    StringBuilder builder;
+
+    for (i = 1; i < argc - 1; i++)
```
Comment (sleepy-monax) [inline applications/coreutils/yes.cpp:9]: I'm not sure about the -1 here. Are we not missing the last arguments.

Comment (Keyboard-Slayer) [reply applications/coreutils/yes.cpp:9]: No, because I add it at line 15

Comment (Keyboard-Slayer) [reply applications/coreutils/yes.cpp:9]: (I did a -1 only to don't have a space at the end)

Comment (sleepy-monax) [reply applications/coreutils/yes.cpp:9]: Sorry, I need to improved at reading other people code 😅 Good work :)

Comment (Keyboard-Slayer) [reply applications/coreutils/yes.cpp:9]: No problem :sweat_smile:

**Pull request merged**